synapse/swarm/opers/basic.py

```diff
@@ -92,30 +92,37 @@ def _oper_lift(self):
 class ByOper(s_opers_common.Oper,ByMix):
     pass
 
-class RangeOper(s_opers_common.CmpOper,ByMix):
+class InOper(s_opers_common.CmpOper,ByMix):
 
     def getByPropValu(self):
-        return 'range',self.args[0],
-
-    def _oper_lift(self):
+        valu = self.kwargs.get('valu')
+        return 'in',self.args[0],valu
 
-        rtup = self.getLiftRange()
+    def getCmpFunc(self):
 
         prop = self.args[0]
-        rtup = self.getLiftRange()
-        limt = self.kwargs.get('limit')
-        ftag = self.kwargs.get('from', s_opers_common.deftag)
+        valu = self.kwargs.get('valu')
 
-        dyntask = gentask('getTufosBy','range', prop, rtup, limit=limt)
+        def cmptufo(tufo):
+            return tufo[1].get(prop) in valu
 
-        # TODO: timeouts
-        for svcfo,tufos in self.query.callByTag(ftag, dyntask):
-            for tufo in tufos:
-                tufo[1]['.from'] = svcfo[0]
-                self.query.add(tufo)
+        return cmptufo
+
+class RangeOper(s_opers_common.CmpOper,ByMix):
+
+    def getByPropValu(self):
+        valu = self.kwargs.get('valu')
+        return 'range',self.args[0],valu
+
+    def getCmpFunc(self):
+
+        prop = self.args[0]
+        valu = self.kwargs.get('valu')
 
-    def getLiftRange(self):
-        pass
+        def cmptufo(tufo):
+            return valu[0] < tufo[1].get(prop) < valu[1]
```

Comment:
our range syntax should probably work like python's range syntax, so
`valu[0] <= tufo[1].get(prop) < valu[1]`